src/modules/uxrce_dds_client/uxrce_dds_client.cpp

```diff
@@ -300,6 +344,17 @@ void UxrceddsClient::run()
 	}
 
 	while (!should_exit()) {
+
+		while (!should_exit() && !_comm) {
+			if (!init()) {
+				// sleep before trying again
+				px4_usleep(1'000'000);
```

Comment:
```suggestion
				px4_usleep(1_s);
```

Reply:
It's kind of a nitpick, but those time literals are an HRT thing, which happens to be microseconds, but could just as easily become nanoseconds someday.